ci/tag-docs.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+### This script should only be run on Improbable's internal build machines.
+### If you don't work at Improbable, this may be interesting as a guide to what software versions we use for our
+### automation, but not much more than that.
+
+set -e -u -x -o pipefail
+
+if [[ -z "$BUILDKITE" ]]; then
+  echo "This script is only to be run on Improbable CI."
+  exit 1
+fi
+
+cd "$(dirname "$0")/../"
+
+source ci/tools.sh
+
+# Tag a version of the docs.
+#
+# Environment variables as arguments:
+#   DOCS_TYPE: "unity"
+#   DOCS_VERSION: "alpha"
+#   DOCS_TARGET: "testing", "staging" or "production"
+
+DOCS_HASH="$(buildkite-agent meta-data get "docs-hash")"
+if [ -z "${DOCS_HASH}" ]; then
+    echo "Expected docs-hash meta-data to be set."
+    exit 1
+fi
+
+if [ "${DOCS_TARGET}" != "production" ]; then
+    DOCS_VERSION="preview-${DOCS_HASH}"
+fi
+
+if [[ "${BUILDKITE_BRANCH}" != "master" && "${DOCS_TARGET}" == "production" ]]; then
+    echo "Docs may only be published to production from master."
+fi
+
+ENVIRONMENT="production"
+
+setup_improbadoc "${ENVIRONMENT}"
+
+improbadoc list \
+    "${DOCS_TYPE}" \
+    --target="${DOCS_TARGET}" \
+    --environment="${ENVIRONMENT}" \
```

Comment:
Environment for improbadoc should always be prod but `target` should change. Reason being that `environment` controls which service to talk to, `testing` and `staging` might not be stable.

Reply:
in which case - can I omit the `environment` argument and trust it will always talk to prod?

Reply:
Yep